CHANGELOG.md

```diff
@@ -3,50 +3,61 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [9.3.4] - 2019-10-21
+## [9.3.4] - 2020-01-26
+
 ### Fixed
 - Be compatible with all in-use cppzmq versions ([#530][pr-530], [#561][pr-561], [#273][i-273], [#499][i-499], [#535][i-535])
+- Fix snprintf detection. Now the `LOG_*` macros work again ([#658][pr-658])
```

Comment:
These kinds of links do not work if we don't add lines like the following in the file (there are already lines like that at the end of the file):
`[pr-658]: https://github.com/tango-controls/cppTango/pull/658`